apps/condo/domains/banking/access/ImportBankTransactionsService.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Generated by `createservice banking.ImportBankTransactionsService`
+ */
+const { get } = require('lodash')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('../../organization/utils/accessSchema')
+
+async function canImportBankTransactions ({ args, authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    const organizationId = get(args, ['data', 'organizationId'])
+    const canManageBankAccounts = checkPermissionInUserOrganizationOrRelatedOrganization(user.id, organizationId, 'canManageBankAccounts')
+    if (!canManageBankAccounts) {
+        return false
+    }
+    const propertyId = get(args, ['data', 'propertyId'])
+    const property = await getById('Property', propertyId)
+    if (!property) {
+        return false
+    }
+    if (property.organization !== organizationId) {
+        return false
+    }
+
+    return true
```

Comment:
It's better to return false by default for access checks. And use return true in some condition.

Reply:
It is a redundant module. Removed it.